corehq/apps/analytics/tasks.py

```diff
@@ -148,6 +148,13 @@ def _send_form_to_hubspot(form_id, webuser, cookies, meta):
 
 
 @task(queue='background_queue', acks_late=True, ignore_result=True)
+def update_hubspot_properties(webuser, properties):
+    vid = _get_user_hubspot_id(webuser)
+    if vid:
+        _track_on_hubspot(webuser, properties)
```

Comment:
From Audrey: 
"anytime we create a new property, it should not bring anyone new to Hubspot. (as anyone who now log in to Hubspot is created as a Hubspot contact through the "CommCare Sign In" form).
And yes we dont want to bring to Hubspot anyone who has ever been a HQ user (at least not yet)"
So, I think we should keep that extra hubspot call for now. Can't really think of other options other than storing something in our db on whether they have been added to HS or not (which seems overkill). In any case, it happens in a task, so at least it isn't slowing down user requests.